### DISCUSSION TITLE
New Dexcom transmitter not connecting

### DISCUSSION THREAD

**Discussion by Schizm1** [NightscoutFoundation/xDrip#2756] [General]
I installed a new dexcom g6 transmitter but xdrip is Stuck in "state is currently unknown. Next connection will update this". I've verified BT code, power cycled and in most updates software/firmware. I've been a xdrip user for many years and this is the first time I've struggled with connection. Any ideas?

**Comment by Navid200** [comment]
Please post a screenshot of the system status page.  You can blackout the transmitter ID if you like.

**Reply by Navid200** [reply]
Please also state what brand and model your phone is, what Android version is on it, and what the two dates are that are printed on your G6 transmitter box.

**Comment by Schizm1** [comment]
Unfortunately I just ripped out the entire transmitter and sensor as I'm
frustrated. So it will now take another 2 hrs before I can do so.

Mike

**Reply by Navid200** [reply]
That's not true.  Please post a screenshot of the system status page.  It has nothing to do with the sensor.

**Comment by Schizm1** [comment]
I will do so

**Reply by Navid200** [reply]
It's passed my bedtime.
Don't waste any more sensors.  We can resolve connectivity first.  
If there is absolutely no connectivity, it is possible that the sensor was damaged and the transmitter has not been activated yet.
You can hold the transmitter in your hand with your thumb on the contacts for a minute or two to activate it.

But, if the transmitter is too old, it is possible that it's battery is dead.  That's why asked for the dates.

**Comment by Schizm1** [comment]
I just replaced a sensor "stopped sensor" added a new sensor but its not
asking for the new code which is different from the old.

**Reply by Navid200** [reply]
I want to help you.  If you do what I asked you, I may be able to.  But, if instead, you tell me you cannot do it, we won't get anywhere.
I need to go to bed now.

Since I didn't get replies from you, I was trying to provide information as much as I can.

The worst thing you can do is to keep wasting sensors.  If I were you, I would stop tapping on stop sensor and start sensor.  it is pointless as long as there is no connectivity.

**Reply by Navid200** [reply]
There is no reason to enter a calibration code because there is no reason to start a sensor.

**Comment by Schizm1** [comment]
Sorry, I will leave the current one in and contact you tomorrow and go
step by step if you're available. It's always been a interesting process
when changing transmitters but always worked itself out with reboots etc.
Have a good night and I look forward to a solution.

**Reply by Navid200** [reply]
I hope that it is working.

For your information, this is what I mean by the status page:  https://navid200.github.io/xDrip/docs/StatusG5G6.html
There is a lot of important info on it that anyone who may be able to help will need in order to be able to help.
You can blackout the transmitter ID if you like.

**Comment by Schizm1** [comment]
Hi, no, it's not working.  Here is a screenshot of my status page

**Comment by Schizm1** [comment]
Hi, I just double checked everything and found it. The new transmitter I'd
had a "0" where I thought it was a "O". They both look identical with the
font on the label and device. I wish they would refrain from using that
number/letter.  Sorry for the bother.

**Comment by spindulel** [comment]
I installed a new dexcom one transmitter and it worked for a day. However yesterday evening it stopped and xdrip is stuck in "state is currently unknown.". I've renewed newest version of xdrip, būt ir didn't help
![Screenshot_2024-03-01-21-04-42-932_com eveningoutpost dexdrip](https://github.com/NightscoutFoundation/xDrip/assets/161881080/00ca273c-2a28-4620-a54d-77d7178303c3)
. This is the first time I've struggled with connection. Any ideas?

**Reply by Navid200** [reply]
Please open a new thread for clarity.  There is no reason to use this particular existing discussion for your problem.

**Comment by Schizm1** [comment]
I just upgraded to the G7. The first sensor was awkward with no
notification of warmup time. It worked after 1/2 hr. 10 days later I just
replace it and xdrip is giving me a 1.8 hr warm-up time when it should be
1/2 hr. Does Xdrip + need something else changed? Doesn't seem right to me.

Mike

**Reply by Navid200** [reply]
Update your xDrip to the latest Nightly.
If you have questions about G7, please open a new thread instead of using an old thread titled transmitter not connecting.

**Comment by Schizm1** [comment]
Is the g7 still in beta? Every time I put in a new sensor (on my 4th) it's
a lottery shoot on what problem will arise. It eventually connects but the
status and displays show no sign of countdown or warmup and I'm always
questioning of its truly working. Will xdrip eventually show a accurate
connection and warmup period with the g7 Or is just me? My last sensor lost
connection and in the upper left corner showed a g6??? I replaced the
sensor and got various messages including a 90 minute warmup but eventually
connected after 30 which it should. I have followed all the instructions
for the g7 off github but still seems to be a very buggy issue.

Mike

**Reply by Navid200** [reply]
No, it's not beta.
Open a new thread and we will help you.
